src/Symfony/Bridge/Doctrine/Tests/SchemaListener/LockStoreSchemaListenerTest.php

```diff
@@ -40,20 +39,4 @@ public function testPostGenerateSchemaLockPdo()
         $subscriber = new LockStoreSchemaListener([$lockStore]);
         $subscriber->postGenerateSchema($event);
     }
-
-    public function testPostGenerateSchemaWithInvalidLockStore()
```

Comment:
Maybe I'm a bit late to the party and missed an important part of the discussion of the previous PR. But… could you recap why we _delete_ a test for a bugfix? I would've expected a _new_ test instead that covers the bug that we want to fix.

Reply:
@MatTheCat wrote done his reasoning here: https://github.com/symfony/symfony/pull/54407#issuecomment-2269032034

Reply:
(I updated the `testPostGenerateSchemaLockPdo` so that it would fail.)

Reply:
You can see `testPostGenerateSchemaWithInvalidLockStore` is wrong because the function creating the generator is static, but its body is using `$this`. That means it only passes because the generator is not started and the `LockStoreSchemaListener` does not get any store.